packages/bbob-plugin-helper/src/index.js

```diff
@@ -52,10 +52,16 @@ const attrValue = (name, value) => {
  * Transforms attrs to html params string
  * @param values
  */
-const attrsToString = values =>
-  Object.keys(values)
+const attrsToString = (values) => {
+  // To avoid some malformed attributes
+  if (undefined === values) {
```

Comment:
I think better to check `typeof values === 'undefined'`, more bulletproof :)

Reply:
It will be better don't call this function at all, and check `values/attrs` in parent call. Or set empty object to `attrs` by default. But i dont know where.